packages/containers/examples/ExampleAction.js

```diff
@@ -12,15 +12,50 @@ const myAction = {
 	},
 };
 
-export default function ExampleAction() {
+const eAction = {
+	onClick: stAction('You clicked me'),
+	payload: {
+		type: 'MY SUPER REDUX ACTION',
+	},
+};
 
+export default function ExampleAction() {
 	return (
 		<div>
 			<IconsProvider />
 			<p>Using names</p>
 			<Action name="menu:first" />
 			<p>Using actions</p>
 			<Action {...myAction} />
+			<p>Using available expression (4 Actions 2 hidden)</p>
```

Comment:
This story is hard to understand. Can you add some description ?
(And change the 2nd `is True expression` by perhaps `should not be displayed: false expression`